docs/config/README.md

```diff
@@ -457,21 +458,28 @@ look at [this example](#with-custom-error-shape).
 
 :::
 
+::: warning
+
+`continuation_symbol` is only supported in bash, zsh, and PowerShell.
+
+:::
+
 ### Options
 
-| Option           | Default             | Description                                                                      |
-| ---------------- | ------------------- | -------------------------------------------------------------------------------- |
-| `format`         | `"$symbol "`        | The format string used before the text input.                                    |
-| `success_symbol` | `"[❯](bold green)"` | The format string used before the text input if the previous command succeeded.  |
-| `error_symbol`   | `"[❯](bold red)"`   | The format string used before the text input if the previous command failed.     |
-| `vicmd_symbol`   | `"[❮](bold green)"` | The format string used before the text input if the shell is in vim normal mode. |
-| `disabled`       | `false`             | Disables the `character` module.                                                 |
+| Option                | Default              | Description                                                                                 |
+| --------------------- | -------------------- | ------------------------------------------------------------------------------------------- |
+| `format`              | `"$symbol "`         | The format string used before the text input.                                               |
+| `success_symbol`      | `"[❯](bold green)"`  | The format string used before the text input if the previous command succeeded.             |
+| `error_symbol`        | `"[❯](bold red)"`    | The format string used before the text input if the previous command failed.                |
+| `continuation_symbol` | `"[❯](bold yellow)"` | The format string used before the text input if the previous command needs to be continued. |
+| `vicmd_symbol`        | `"[❮](bold green)"`  | The format string used before the text input if the shell is in vim normal mode.            |
+| `disabled`            | `false`              | Disables the `character` module.                                                            |
 
 ### Variables
 
-| Variable | Example | Description                                                           |
-| -------- | ------- | --------------------------------------------------------------------- |
-| symbol   |         | A mirror of either `success_symbol`, `error_symbol` or `vicmd_symbol` |
+| Variable | Example | Description                                                                                   |
+| -------- | ------- | --------------------------------------------------------------------------------------------- |
+| symbol   |         | A mirror of either `success_symbol`, `error_symbol`, `continuation_symbol`, or `vicmd_symbol` |
```

Comment:
Why would this mirror `continuation_symbol`?

Reply:
I thought that the value of `symbol` would be set to the value of either `success_symbol`, `error_symbol`, `continuation_symbol`, or `vicmd_symbol`, depending on the last entered command.

As the continuation prompt is going to be static and will be moved out of the character module anyway, I'll fix this by the next commit.